app/actions/remote/entry/gql_common.ts

```diff
@@ -0,0 +1,307 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Database} from '@nozbe/watermelondb';
+
+import {markChannelAsRead, MyChannelsRequest} from '@actions/remote/channel';
+import {fetchGroupsForMember} from '@actions/remote/groups';
+import {fetchPostsForChannel, fetchPostsForUnreadChannels} from '@actions/remote/post';
+import {MyTeamsRequest} from '@actions/remote/team';
+import {fetchNewThreads} from '@actions/remote/thread';
+import {MyUserRequest, updateAllUsersSince} from '@actions/remote/user';
+import {gqlEntry, gqlEntryChannels, gqlOtherChannels} from '@client/graphQL/entry';
+import {Preferences} from '@constants';
+import DatabaseManager from '@database/manager';
+import {getPreferenceValue} from '@helpers/api/preference';
+import {selectDefaultTeam} from '@helpers/api/team';
+import {queryAllChannels, queryAllChannelsForTeam} from '@queries/servers/channel';
+import {prepareModels, truncateCrtRelatedTables} from '@queries/servers/entry';
+import {getHasCRTChanged} from '@queries/servers/preference';
+import {prepareCommonSystemValues} from '@queries/servers/system';
+import {addChannelToTeamHistory, addTeamToTeamHistory, queryMyTeams} from '@queries/servers/team';
+import {selectDefaultChannelForTeam} from '@utils/channel';
+import {filterAndTransformRoles, getMemberChannelsFromGQLQuery, getMemberTeamsFromGQLQuery, gqlToClientChannelMembership, gqlToClientPreference, gqlToClientSidebarCategory, gqlToClientTeamMembership, gqlToClientUser} from '@utils/graphql';
+import {isTablet} from '@utils/helpers';
+import {processIsCRTEnabled} from '@utils/thread';
+
+import {teamsToRemove} from './common';
+
+import type ClientError from '@client/rest/error';
+import type ChannelModel from '@typings/database/models/servers/channel';
+import type TeamModel from '@typings/database/models/servers/team';
+
+export async function deferredAppEntryGraphQLActions(
+    serverUrl: string,
+    since: number,
+    meData: MyUserRequest,
+    teamData: MyTeamsRequest,
+    chData: MyChannelsRequest | undefined,
+    isTabletDevice: boolean,
+    initialTeamId?: string,
+    initialChannelId?: string,
+    isCRTEnabled = false,
+    syncDatabase?: boolean,
+) {
+    const operator = DatabaseManager.serverDatabases[serverUrl]?.operator;
+    if (!operator) {
+        return {error: `${serverUrl} database not found`};
+    }
+    const {database} = operator;
+
+    // defer fetching posts for initial channel
+    if (initialChannelId && isTabletDevice) {
+        fetchPostsForChannel(serverUrl, initialChannelId);
+        markChannelAsRead(serverUrl, initialChannelId);
+    }
+
+    if (chData?.channels?.length && chData.memberships?.length) {
+        // defer fetching posts for unread channels on initial team
+        fetchPostsForUnreadChannels(serverUrl, chData.channels, chData.memberships, initialChannelId);
```

Comment:
On the Gekidou branch, we had a fix from PR #6453  to delay the fetchPostsForUnreadChannels call.  Though it was a fix,  we should check if this is happening on the 'GraphQL' flow as well.